src/moscot/_types.py

```diff
@@ -51,8 +51,9 @@
     "sequential",
     "star",
     "external_star",
+    "pairwise",
```

Comment:
`pairwise` was removed in the constants , `src/moscot/_constants.py`, right? why?

Reply:
Because pairwise policy doesn't exist anymore; this should also be removed here.